core/domain/story_domain_test.py

```diff
@@ -344,6 +344,7 @@ def test_defaults(self):
             'title': 'Title',
             'thumbnail_filename': None,
             'thumbnail_bg_color': None,
+            'thumbnail_size_in_bytes': None,
```

Comment:
@seanlip 
Do you think we need tests for the changes we did in `story_domain.Story.update_thumbnail_filename()` method ? 
(Asking this because, the PR is approved and close to being merged -- I am not sure if the backend requires this test)

Reply:
Yes, try to write tests for all behaviours.

Reply:
Done